app/src/components/EmptyList.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Text, View } from 'react-native'
+
+import EmptyWallet from '../assets/img/emptyWallet.svg'
+import { useTheme, testIdWithKey, Button, ButtonType, useStore } from 'aries-bifold'
+import { BCState, BCDispatchAction } from '../store'
+export interface EmptyListProps {
```

Comment:
Space above.

Reply:
I think the linter in bifold likes to have an empty line between import groups. We aren't using the same linter as bifold but I figured I'd keep it consistent